django/db/models/sql/compiler.py

```diff
@@ -903,17 +905,102 @@ def __init__(self, *args, **kwargs):
         self.return_id = False
         super(SQLInsertCompiler, self).__init__(*args, **kwargs)
 
-    def placeholder(self, field, val):
+    def field_as_sql(self, field, val):
+        """
+        Take a field, and a value intended to be saved on that field, and
+        return placeholder SQL and accompanying params. Checks for raw values,
+        expressions and fields with get_placeholder() defined in that order.
+        """
+
         if field is None:
```

Comment:
Since the field can be `None`, would it be useful documenting in the above docstring that special case?